wps-cloudformation-template.yaml

```diff
@@ -60,6 +57,15 @@ Parameters:
   workerRetryIntervalMs:
     Type: String
     Default: '60000'
+  acmCertificate:
+    Type: String
+    Default: ''
+  hostedZoneName:
+    Type: String
+    Default: 'aodn.org.au.'
+  domainName:
```

Comment:
I would have treated domain name similar to the bucket name because: 

1. It makes more sense to me to move it under the mapping as it is unlikely that the domain name is going to change.
2. To make sure that the right domain name is associated to an environment. 

```yaml
Mappings:
  EnvironmentMap:
    production:
      BucketName: "aws-wps-production"
      DomainName: "wps"
    systest:
      BucketName: "aws-wps-systest"
      DomainName: "wps-systest"
    sandbox:
      BucketName: "aws-wps-sandbox"
      DomainName: "wps-sandbox"
    dev:
      BucketName: "aws-wps-dev"
      DomainName: "wps-dev"
```

Reply:
@lwgordonimos  What are your thoughts on this one?